### ISSUE TITLE
error McuClass::begin

### ISSUE THREAD

**Issue by jamesgbahr** [ShotokuTech/HeltecLoRaButtonRelay#1]
no matching function for call to 'McuClass::begin()'

**Comment by ShotokuTech** [comment]
Oddly, I found using the latest board manager URL and board support, I encountered several compile errors.
I remove the board support and replaced the board manager URL with this one:
https://github.com/Heltec-Aaron-Lee/WiFi_Kit_series/releases/download/0.0.7/package_heltec_esp32_index.json
Then I installed board support 0.0.7 and I could compile without issue. Make sure you have selected the correct board as well. There are big differences from WiFi Lora 32 V2 and V3.
https://github.com/Heltec-Aaron-Lee/WiFi_Kit_series/issues/211

**Comment by ShotokuTech** [comment]
I finally reproduced your error. Now I got to try to fix it. The problem is not with my code. It is with the Heltec platform.

**Comment by ShotokuTech** [comment]
I spent way too much time on this one! LOL.
What it boils down to is using the latest board manager URL: https://resource.heltec.cn/download/package_heltec_esp32_index.json
Install board manager package: Heltec ESP32 Series Dev-Boards 3.0.2
Install library: Heltec ESP32 Dev-Boards
Then in the code, find: Mcu.begin();
And change it to: Mcu.begin(HELTEC_BOARD,SLOW_CLK_TPYE);
I was able to compile the code after that.
